packages/expo-image/CHANGELOG.md

```diff
@@ -162,3 +162,5 @@ _This version does not introduce any user-facing changes._
 ### 🎉 New features
 
 - Initial release 🥳
+
+- Added all the extra props to expo-image on web ([#22340](https://github.com/expo/expo/pull/22340) by [@makkarMeenu](https://github.com/makkarMeenu))
```

Comment:
please move to the unpublished section on the top of the changelog and reword so it's clear that it does prop spread :) 
`Use prop spreading on web to pass all unused props to the native image component`

Reply:
Please find the changes in [this](https://github.com/expo/expo/pull/22340/commits/687ab7b5aca364d1f701ed81176764944ff27a8a) commit